esx_service/utils/auth_data.py

```diff
@@ -576,7 +638,9 @@ def connect(self):
 
         if not need_create_table:
             if os.path.isfile(self.db_path) and self.need_upgrade_db():
-                self.handle_upgrade_db()
+                error = self.handle_upgrade_db()
+                if error:
+                    raise DbConnectionError(self.db_path)
```

Comment:
No. These exceptions are captured and printed neatly to the user as the command output. Mentioning one such example in PR description.